caf-audit-service-container/README.md

```diff
@@ -0,0 +1,35 @@
+# Audit Web Service Container
+
+This is a docker container for the [CAF Audit Web Service](https://github.com/CAFAudit/audit-service/tree/develop/caf-audit-service). It consists of a Tomcat web server that connects to Elasticsearch. It uses the java:8 base image. [Audit-service-deploy](https://github.com/CAFAudit/audit-service-deploy) can be used to deploy this container on Docker.
```

Comment:
[Audit-service-deploy](https://github.com/CAFAudit/audit-service-deploy)  
Changed to:  
[Audit Service Deploy](https://github.com/CAFAudit/audit-service-deploy)